README.md

```diff
@@ -44,6 +48,7 @@ Veneur expects to have a config file supplied via `-f PATH`. The include `exampl
 
 * `api_hostname` - The Datadog API URL to post to. Probably `https://app.datadoghq.com`.
 * `metric_max_length` - How big a buffer to allocate for incoming metric lengths. Metrics longer than this will get truncated!
+* `flush_max_per_body` - how many metrics to include in each JSON body POSTed to Datadog. Veneur will POST multiple bodies in parallel if it goes over this limit. A value around 100k-150k is recommended; in practice we've seen Datadog reject bodies over about 195k.
```

Comment:
What do you think of setting this to something closer to 10k metrics? Let me explain why I think this might be good!

Here's a sample of flush stats from hurricane1.nw, which is one of the most intense boxes for raw packet count:
```
2016-06-14 13:47:27 UTC | INFO | dd.dogstatsd | dogstatsd(dogstatsd.py:210) | Flush #50194: flushed 9112 metrics, 0 events, and 1 service check run
2016-06-14 13:47:37 UTC | INFO | dd.dogstatsd | dogstatsd(dogstatsd.py:210) | Flush #50195: flushed 8414 metrics, 0 events, and 2 service check runs
```

That's ~10k per flush.  Here's an apibox:
```
2016-06-14 13:55:07 UTC | INFO | dd.dogstatsd | dogstatsd(dogstatsd.py:210) | Flush #6861: flushed 4294 metrics, 0 events, and 3 service check runs
2016-06-14 13:55:17 UTC | INFO | dd.dogstatsd | dogstatsd(dogstatsd.py:210) | Flush #6862: flushed 4608 metrics, 0 events, and 3 service check runs
2016-06-14 13:55:27 UTC | INFO | dd.dogstatsd | dogstatsd(dogstatsd.py:210) | Flush #6863: flushed 3441 metrics, 0 events, and 2 service check runs
```

Given that Datadog seeems to be optimized for these individual hosts to POST their work, I think we should endeavor to get Veneur as close to that behavior as is feasible. Since we're seeing around 200k metrics we can expect around 20 goroutines here. That kinda seems like a lot to my thread-brain, but gouroutines should be ok with this, yeah? 😅 

If DD deals better with smaller body sizes this could improve our overall POST time. Unfortunately as you point out we won't really know till we're in prod. But we could pretty easily write a "load" script for QA that generates a bunch of timers (since timers expand to N more metrics) and make each one a random name so we get a big corpus.

Also this means that we're no longer an oddly shaped problem to DD. They are used to us having a single host (one of the proxies) that [send ~200 requests a second](https://stripe.datadoghq.com/dash/68384/datadog-stats-proxy-overview?live=true&page=0&is_auto=false&from_ts=1465909201958&to_ts=1465912801958&tile_size=m&fullscreen=80074972). So adding another box to that and posting an average of 2req/s (~20 every 10s) seems peaceful.